README.md

```diff
@@ -10,43 +10,220 @@
   <img alt="License: MIT" src="https://img.shields.io/badge/License-MIT-purple.svg">
 </a>
 
-A package for building highly interactive user interfaces in pure Python inspired by
-[ReactJS](https://reactjs.org/).
+`django-idom` allows you to integrate [IDOM](https://github.com/idom-team/idom) into
+Django applications. IDOM being a package for building responsive user interfaces in
+pure Python which is inspired by [ReactJS](https://reactjs.org/). For more information
+on IDOM refer to [its documentation](https://idom-docs.herokuapp.com).
 
-**Be sure to [read the IDOM Documentation](https://idom-docs.herokuapp.com)!**
-
-If you have ideas or find a bug, be sure to post an
-[issue](https://github.com/idom-team/django-idom/issues)
-or create a
-[pull request](https://github.com/idom-team/django-idom/pulls). Thanks in advance!
-
-<h3>
-  <a
-    target="_blank"
-    href="https://mybinder.org/v2/gh/idom-team/idom-jupyter/main?filepath=notebooks%2Fintroduction.ipynb"
-  >
-    Try it Now
-    <img alt="Binder" valign="bottom" height="25px"
-    src="https://mybinder.org/badge_logo.svg"
-    />
-  </a>
-</h3>
+**You can try IDOM now in a Jupyter Notebook:**
+<a
+  target="_blank"
+  href="https://mybinder.org/v2/gh/idom-team/idom-jupyter/main?filepath=notebooks%2Fintroduction.ipynb">
+  <img
+    alt="Binder"
+    valign="bottom"
+    height="21px"
+    src="https://mybinder.org/badge_logo.svg"/>
+</a>
 
-Click the badge above to get started! It will take you to a [Jupyter Notebooks](https://jupyter.org/)
-hosted by [Binder](https://mybinder.org/) with some great examples.
 
-### Or Install it Now
+# Install Django IDOM
 
 ```bash
 pip install django-idom
 ```
 
 # Django Integration
 
-This version of IDOM can be directly integrated into Django. For example
+To integrate IDOM into your application you'll need to modify or add the following files to `your_app`:
+
+```
+your_app/
```

Comment:
I updated the structure in the README. Let me know if that needs more work.

Reply:
Apps created by Django's default `manage.py startapp <app_name>` command have the following files: `admin.py`, `apps.py`, `models.py`, `tests.py`, and `views.py`.

Although `urls.py` is a extremely common convention it's not a default.

In this example I would put in all of Django's default files, remove `urls.py` (since it's technically not a default), and then add in your two custom app files.

Reply:
I think adding files that aren't discussed below might be confusing. I really wanted to just highlight what was important rather than show what all the files in a working app might be.

Reply:
Sounds good. I still recommend changing `sub_app/` to `example_app/` or `my_app/`. Especially in this example, as the project root isn't an app. 

In `django_idom`'s repository structure the project root is both an app and the root, so `sub_app/` would be appropriate in that case. But that doesn't hold true to the example project structure in this readme.